### ISSUE TITLE
Screen reader users should be made aware of route changes

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-react-seed#55]
Generally, we want to ensure that after a new view is loaded, screen reader users are informed and adequately oriented so that they can easily find and consume the new content.

For us to do this, the following needs to be addressed;

- Page `<title>` needs to be updated per page
- Ensure each page contains a main heading
- Ensure keyboard focus is managed and not lost between page changes

Currently, after selecting an item from the primary navigation, nothing relevant to the page change is announced and it's verified that the user is still focused on the link. Let's improve this so that some basic information is provided and ideally send focus to the new content once it's available.
